spec/sec-operations-on-objects-patch.html

```diff
@@ -0,0 +1,25 @@
+<emu-clause id="sec-operations-on-objects">
+  <h1>Operations on Objects</h1>
+  <ins>
+    <emu-clause id="sec-getoption" aoid="GetOption">
+      <h1>GetOption ( _options_, _property_, _type_, _values_, _fallback_ )</h1>
+      <emu-note>This is the same operation as <a href="https://tc39.es/ecma402/#sec-getoption">GetOption</a> in ECMA-402.</emu-note>
+      <p>
+        The abstract operation GetOption extracts the value of the property named _property_ from the provided _options_ object, converts it to the required _type_, checks whether it is one of a List of allowed _values_, and fills in a _fallback_ value if necessary. If _values_ is *undefined*, there is no fixed set of values and any is permitted.
+      </p>
+      <emu-alg>
+        1. Let _value_ be ? Get(_options_, _property_).
+        1. If _value_ is not *undefined*, then
+          1. Assert: _type_ is `"boolean"` or `"string"`.
```

Comment:
```suggestion
```
i think this assertion should apply even if value is undefined, and appear at the top.